Gemfile

```diff
@@ -39,7 +43,7 @@ group :dependencies do
   gem 'rdiscount', '2.2.0.1'
 
   # Process manager for applications with multiple components
-  gem 'foreman', '~> 0.85.0'
+  gem 'foreman', git: 'https://github.com/alaxalves/foreman.git', branch: 'master'
```

Comment:
Had to do this to keep the `thor` gem compatibility both with `rails` and `foreman` gems. Done [here](https://github.com/alaxalves/foreman/commit/ac1c0919f8a8f5c81112bc1bb9b41a2565d56202)